mode/type/standalone/repository/provider/jdbc/mysql/src/test/java/org/apache/shardingsphere/mode/repository/standalone/jdbc/mysql/MysqlJDBCRepositoryProviderTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.mode.repository.standalone.jdbc.mysql;
+
+import org.apache.shardingsphere.mode.repository.standalone.jdbc.JDBCRepository;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Properties;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class MysqlJDBCRepositoryProviderTest {
+    
+    private final JDBCRepository repository = new JDBCRepository();
+    
+    @Before
+    public void setUp() {
+        Properties props = new Properties();
+        props.setProperty("jdbc_url", "jdbc:tc:mysql:8.0.31:///config");
+        props.setProperty("username", "sa");
+        props.setProperty("password", "");
+        props.setProperty("provider", "MYSQL");
+        repository.init(props);
+    }
+    
+    @After
+    public void stop() {
+        repository.close();
+    }
+    
+    @Test
+    public void assertPersistAndGet() {
+        repository.persist("/testPath/test1", "test1_content");
+        assertThat(repository.getDirectly("/testPath/test1"), is("test1_content"));
+        repository.persist("/testPath/test1", "modify_content");
+        assertThat(repository.getDirectly("/testPath/test1"), is("modify_content"));
+    }
+    
+    @Test
+    public void assertPersistAndGetChildrenKeys() {
+        repository.persist("/testPath/test1", "test1_content");
+        repository.persist("/testPath/test2", "test2_content");
+        List<String> childrenKeys = repository.getChildrenKeys("/testPath");
+        assertThat(childrenKeys.get(0), is("test2"));
+        assertThat(childrenKeys.get(1), is("test1"));
+    }
```

Comment:
- It looks like MySQL's ResultSet is sorted differently by default than H2Database, which I think makes sense.

Reply:
- I added order by restriction to selectByParentKeySQL.